### PR TITLE
host: Extract RecentCards service

### PR DESCRIPTION
**Pull request by backspace**
This covers some of the operator mode container refactoring ideas.

### CODE REVIEW

**Review by lukemelia — APPROVED** [review]
Better 👍

File: packages/host/app/components/search-sheet/index.gts [hunk 1]
```diff
@@ -208,7 +210,7 @@ export default class SearchSheet extends Component<Signature> {
   @cached
   get orderedRecentCards() {
     // Most recently added first
-    return [...this.operatorModeStateService.recentCards].reverse();
+    return [...this.recentCards.recentCards].reverse();
```
Comment (jurgenwerk) [inline packages/host/app/components/search-sheet/index.gts]: feels a bit awkward to say `recentCards.recentCards`, maybe adjust to `recentCardsService.recentCards`?

File: packages/host/app/services/operator-mode-state-service.ts [hunk 2]
```diff
@@ -79,13 +79,13 @@ export default class OperatorModeStateService extends Service {
...
     codeSelection: new TrackedObject({}),
   });
-  @tracked recentCards = new TrackedArray<CardDef>([]);
 
   private cachedRealmURL: URL | null = null;
 
   @service declare cardService: CardService;
   @service declare loaderService: LoaderService;
   @service declare messageService: MessageService;
+  @service declare recentCards: RecentCardsService;
```
Comment (jurgenwerk) [inline packages/host/app/services/operator-mode-state-service.ts]: All other service names have the `*Service` postfix

**Review by jurgenwerk — APPROVED** [review]
Looks good, minor service nomenclature comment

File: packages/host/app/components/search-sheet/index.gts (hunk 1 above)
Comment (habdelra) [reply packages/host/app/components/search-sheet/index.gts]: the challenge here is that the service file name is `recent-cards.ts`. to add the prefix we'd have to change the file name. but i think that your reasoning is why all the other services have the `-service` suffix.

**Review by habdelra — APPROVED** [review]

**Comment by backspace** [comment]
ya I don’t love the redundant `…Service` suffix on all the service names, but I’ll change this one for consistency 😬

**Pull request merged**